onnxruntime/test/server/unit_tests/server_configuration_test.cc

```diff
@@ -23,6 +25,8 @@ TEST(ConfigParsingTests, AllArgs) {
   Result res = config.ParseInput(11, test_argv);
```

Comment:
>test_argv [](start = 37, length = 9)

Is ParseInput modifying the input array args? Does it insert zeros like strtok? Can this be fixed? It is potentially a thread-safety issue.

Reply:
I don't think it does. It just parsing them with `po::store(po::command_line_parser(argc, argv).options(desc).run(), vm);  // can throw`